pkg/tracer/event_test.go

```diff
@@ -11,22 +11,22 @@ import (
 
 var (
 	testConn = ConnectionStats{
-		Pid:       123,
-		Type:      1,
-		Family:    0,
-		Source:    "192.168.0.1",
-		Dest:      "192.168.0.103",
-		SPort:     123,
-		DPort:     35000,
-		SendBytes: 123123,
-		RecvBytes: 312312,
+		Pid:        123,
+		Type:       1,
+		Family:     0,
```

Comment:
Can we use the const values for `Type` and `Family` to avoid confusion of the types and make it easier to maintain down the line

Reply:
There's a few instance in this file where it uses the int value, the rest looks good